src/table.mjs

```diff
@@ -231,3 +271,91 @@ function likeOperand(operand) {
   return {...operand, value: `%${operand.value}%`};
 }
 
+// This function applies table cell operations to an in-memory table (array of objects).
+export function __table(source, operations) {
+  let {schema, columns} = source;
+  for (const {type, operands} of operations.filter) {
+    const column = operands.find(({type}) => type === "column").value;
+    const resolved = operands.filter(({type}) => type === "resolved");
+    switch (type) {
+      case "eq": {
+        const [{value}] = resolved;
+        source = source.filter((d) => d[column] === value);
```

Comment:
We’re using strict equality here, but the resolved value will always be a string (in our UI), whereas the type of the column might be a number. I think this means we’ll have to use loose equality here instead… and I’m not sure how we’ll handle dates.